src/utils/normalize-otherAttribute-contact.ts

```diff
@@ -0,0 +1,52 @@
+import cloneDeep from 'lodash/cloneDeep';
+import forEach from 'lodash/forEach';
+import { denormalize } from '../normalize';
+import { ContactInputRequest } from '../normalize/entities';
+
+export function createContactBody(data: any) {
+	const { attributes, ...rest } = data;
+	const contactAttrs = <Object[]>[];
+
+	forEach(attributes, (val, key) =>
+		key !== 'other'
+			? contactAttrs.push({
+					name: key,
+					[key === 'image' ? 'aid' : 'content']: val
+			  })
+			: forEach(val, (otherValue: any, index: any) =>
+					contactAttrs.push({
+						name: 'custom' + (index + 1),
+						_content: otherValue
+					})
+			  )
+	);
+	return {
+		cn: {
```

Comment:
We can skip the extra `...` here since `denormalize` will already return a new object:

```js
return {
  cn: denormalize(ContactInputRequest)({
    ...rest,
    attributes: contactAttrs
  })
};
```

Reply:
Removed.